server/Movieflix/src/main/java/rest/module/Movieflix/Entity/Administrators.java

```diff
@@ -0,0 +1,89 @@
+package rest.module.Movieflix.Entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity
+@Table
+@NamedQueries({ 
+		@NamedQuery(name = "Administrators.findAll", query = "SELECT e FROM Administrators e ORDER BY e.email ASC"),
+		@NamedQuery(name = "Administrators.findByEmail", query = "SELECT e FROM Administrators e WHERE e.email=:pEmail") 
+})
+public class Administrators {
```

Comment:
Don't think we need a different entity for `Admin`.
User entity can be used for both. The only thing different is the role.